clang_format/BUILD

```diff
@@ -0,0 +1,43 @@
+load(":choose_clang_format.bzl", "choose_clang_format")
+
+filegroup(
+    name = "clang_format_config_default",
+    srcs = [ "clang-format" ],
+)
+
+label_flag(
+    name = "clang_format_config",
```

Comment:
Ok it seem like this isn't quite working properly. If i override this flag with the `.clang-format` from the project then it works as expected (no errors). Otherwise it formats the file according to the default rules.